README.md

```diff
@@ -53,6 +53,10 @@ This will prompt for parameters for project initialization. Some of these parame
    Otherwise, user can leave it blank to use the name of the current project as root directory name.
  * ``cloud``: Cloud provider you use with Databricks (AWS, Azure, or GCP)
  * ``cicd_platform`` : CI/CD platform of choice (GitHub Actions or GitHub Actions for GitHub Enterprise Servers or Azure DevOps)
+ * ``framework``: Choose between Delta Tables (default), Feature Store, and MLflow Recipes.
+   - If ``Delta Tables`` is selected, will provide the default stack using Delta Tables for data input and output.
```

Comment:
We've talked a lot offline. Let's bring the discussion online.

There are two parts of the problem:
1. Given the new option of non-feature-store non-recipe training, what will be the user facing questions to ask users? 2 yes/no questions or 1 enum selection?
2. For internal implementation, do we want to use 2 variable or one enum? 

### For question 2, 
I'm convinced that it will be cleaner in code if we use 1 enum to switch between feature-store / recipe / no-fs-no-recipe.

### For question 1, 
the tradeoffs are:

- If use two questions to ask if recipes / fs is enabled, 
    - recipe(yes) + fs(yes) is not a valid combination. 
    - one more question to ask users
    - (Maybe) easier to deprecate recipe from mlops stacks in the future?
- If use a single question
    - We have to find a better name of  `Delta Tables` because essentially all three option use delta table and input and output
        - any name suggestion? simple training notebook? 
    - fs and recipes are not the same level of option. fs uses different batch inference code, there's no model validation support for fs. mlflow recipe is only for training notebook.

cc @vladimirk-db  @dbczumar @sebastiaanboer-db @zhe-db

Reply:
Thanks for the write-up @mingyu89 !

I think if we can find a better name for `Delta Tables` and provide an accurate description, it would be the simplest to reason about with a single question. Asking two questions and then throwing on an invalid combo is not a great user experience. Plus it will inevitably lead to questions like "When will FS work with MLflow recipes?" 😂

Reply:
Prefer option 1 for question 1.
use mlflow recipes [no is default]
use feature store [no is default]

If we can do conditional cookiecutter where if a user selects mlflow recipes, then the feature store question is omitted, it would be ideal. But it's not strictly required.

Reply:
Since recipe(yes) + fs(yes) is not a valid combination, do we want to skip the second question for fs if user give yes to recipes?

Reply:
Updated the code, conditional variable does not seem to be possible at this time.

Reply:
Yeah